go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -401,9 +403,8 @@ func (vc *VitessCluster) AddTablet(t testing.TB, cell *Cell, keyspace *Keyspace,
 	tablet := &Tablet{}
 
 	options := []string{
-		"--queryserver-config-schema-reload-time", "5",
-		"--enable-lag-throttler",
-		"--heartbeat_enable",
```

Comment:
We don't want to remove this, do we? This may be what's causing the test failures.